sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -1,19 +1,19 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 12.0.0-beta.5 (2020-11-13)
 
 ### Breaking Changes
 
-* Move header `azure/storage/files/datalake/datalake.hpp` to `azure/storage/files/datalake.hpp`
+* Moved header `azure/storage/files/datalake/datalake.hpp` to `azure/storage/files/datalake.hpp`
+* Moved returning model types and related functions in `Azure::Storage::Files::DataLake` to `Azure::Storage::Files::DataLake::Models`, and other code into `Azure::Storage::Files::DataLake::Details`.
+* Renamed `Azure::Storage::Files::DataLake::ServiceClient` to `Azure::Storage::Files::DataLake::DataLakeServiceClient`.
 
 ## 1.0.0-beta.4 (2020-10-16)
 
 ### Bug Fixes
 
 * Unencoded FileSystem/File/Path/Directory name is now encoded.
 
-## 1.0.0-beta.3 (2020-10-13)
-
```

Comment:
Remind me what's happening here and do we need to make a similar change in https://github.com/Azure/azure-sdk-for-cpp/pull/974?

cc @Jinming-Hu

Reply:
Pinged you in teams. So we got rid of beta.3 because of the data integrity issue, now the customer is not able to see that in releases/tags page, and it would be nice to make it consistent in changelog.